usaspending_api/common/helpers/list_helpers.py

```diff
@@ -0,0 +1,11 @@
+
+
+def safe_add(list1, list2):
```

Comment:
It might be easier to use `sets` to combine the distinct values from each list of ids. In fact, this helper function wouldn't be necessary and the logic could be seamlessly rolled into the `handle()` function